.github/workflows/ci-cd.yml

```
@@ -177,8 +177,9 @@ jobs:
         python-version: ${{ matrix.pyver }}
     - name: Get pip cache dir
       id: pip-cache
+      shell: bash
       run: |
-        echo "::set-output name=dir::$(pip cache dir)"    # - name: Cache
+        echo "dir=$(pip cache dir)" >> "$GITHUB_OUTPUT"
```

*(optional, non-blocking)* my preferred style is
```suggestion
        echo "dir=$(pip cache dir)" >> "${GITHUB_OUTPUT}"
```